esp.lic

```diff
@@ -0,0 +1,13 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#esp
+=end
+custom_require.call(%w[common])
+
+bput("esp listen general", "With a moment of focus")
```

Comment:
Should the channels be in base, and overwritable, or does everyone tune most of these channels?

Reply:
Agree, all the channels should be listed in `base.yaml` with option for users to override the list in their yaml.

Something like:
```yaml
esp_listen_channels:
  - general
  - trade
  - personal
  ...

esp_default_channel: general
```

Any channels NOT listed in `esp_listen_channels` would explicitly be turned off when the script runs.

The last line in this script, `bput("esp send general", "You prepare to project")`, would use the setting `esp_default_channel`